src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImpl.java

```diff
@@ -50,9 +50,16 @@ public void performRequestDetailLineValidation(ConcurRequestExtractRequestDetail
         if (requestDetailFileLineIsCashAdvance(detailFileLine)) {
             boolean lineValidationPassed = true;
             lineValidationPassed =  requestedCashAdvanceHasNotBeenClonedInFile(detailFileLine, uniqueRequestIdsInFile);
+            LOG.info("requestedCashAdvanceHasNotBeenClonedInFile = " + lineValidationPassed);
+            
             lineValidationPassed &= requestedCashAdvanceHasNotBeenUsedInExpenseReport(detailFileLine);
+            LOG.info("requestedCashAdvanceHasNotBeenUsedInExpenseReport = " + lineValidationPassed);
```

Comment:
My intent was to see/log at what point the line validation initially/first fails. 
The state of the flags in the validation result on each detail line should show the the remaining validation failures/successes.

Reply:
Ok, thanks for clarifying. I think it would be helpful to clean up some of these logging statements so that the intent of the failure logging is clearer; however, since there's already a pre-existing area with such a setup, I'm fine with deferring its cleanup to a future user story.